miqa/core/tasks.py

```diff
@@ -106,41 +108,41 @@ def perform_import(import_dict, project_id):
     Scan.objects.bulk_create(new_scans)
     Frame.objects.bulk_create(new_frames)
 
-    evaluate_data.delay([frame.id for frame in new_frames], project.id)
+    frames_by_project = {}
+    for frame in new_frames:
+        project_id = str(frame.scan.experiment.project.id)
+        if project_id not in frames_by_project:
+            frames_by_project[project_id] = []
+        frames_by_project[project_id].append(str(frame.id))
```

Comment:
It would be fine, and slightly preferable, to pass around `UUID` types rather than converting them to strings, but that's out of scope of this PR. We can do a follow-up one to refactor that at some point.

Reply:
At least for now, these two new `str` casts can be removed without any trouble. Removed in [1b8c6fb](https://github.com/OpenImaging/miqa/pull/294/commits/1b8c6fb6cc9195bc005447d82623966ada82bc75)